src/components/NewResponses/InputQuestionCard.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <question-card-base :title="title" :is-required="isRequired">
+    <div v-if="questionType === QuestionType.Number">
+      <InputNumber
+        :placeholder="0"
```

Comment:
```suggestion
        placeholder="0"
```
![image](https://user-images.githubusercontent.com/48485650/185765932-5e2e11b2-9748-4eca-bf4d-eaa7d81f5e91.png)